compiler/luci-interpreter/src/kernels/GRU.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "kernels/GRU.h"
+
+#include "kernels/Utils.h"
+
+#include "PALFullyConnected.h"
+#include "PALGRU.h"
+
+namespace luci_interpreter
+{
+namespace kernels
+{
+GRU::GRU(const Tensor *input, const Tensor *hidden_hidden, const Tensor *hidden_hidden_bias,
+         const Tensor *hidden_input, const Tensor *hidden_input_bias, const Tensor *state,
+         Tensor *output, const GRUParams &params)
+  : KernelWithParams<GRUParams>(
+      {input, hidden_hidden, hidden_hidden_bias, hidden_input, hidden_input_bias, state}, {output},
+      params)
+{
+}
+
+void GRU::configure()
+{
+  auto hidden_hidden_shape = getTensorShape(hidden_hidden());
+  auto hidden_input_shape = getTensorShape(hidden_input());
+  LUCI_INTERPRETER_CHECK(hidden_hidden_shape.Dims(0) == hidden_input_shape.Dims(0));
+
+  auto output_shape = getTensorShape(output());
+  auto state_shape = getTensorShape(state());
```

Comment:
May I ask what `output_shape`, `state_shape`  were added for?

Reply:
I will updated soon ( it forgot to remove this in the draft)